logbook-servlet/src/main/java/org/zalando/logbook/servlet/LocalResponse.java

```diff
@@ -54,11 +54,6 @@ public String getProtocolVersion() {
     }
 
     @Override
-    public String getContentType() {
-        return Optional.ofNullable(super.getContentType()).orElse("");
```

Comment:
The fix is not obvious from the first sight, I suppose it changes `""` to `null`. Could you add a test maybe?